File: Step1/Basics/Data-Types/Solution.java
class Solution {
    static int dataTypeSize(String str) {
        // code here
        if(str.equals("Character")){
            return 2;
        }
        else if(str.equals("Integer")){
            return 4;
        }
        else if(str.equals("Long")){
            return 8;
        }
        else if(str.equals("Float")){
            return 4;
        }
        else if(str.equals("Double")){
            return 8;
        }
        else {
            return -1;
        }
    }
}

File: Step1/Basics/Data-Types/Solution.cpp
class Solution {
  public:
    int dataTypeSize(string str) {
        // your code here
        if(str == "Character"){
            return 1;
        }
        else if(str == "Integer"){
            return 4;
        }
        else if(str == "Long"){
            return 8;
        }
        else if(str == "Float"){
            return 4;
        }
        else if(str == "Double"){
            return 8;
        }
        else {
            return -1;
        }
    }
};
